README.md

```diff
@@ -416,6 +416,17 @@ Export the key ID as a [variable](https://stackoverflow.com/questions/1158091/de
 $ export KEYID=0xFF3E7D88647EBCDB
 ```
 
+# Sign with an existing key (if you have one)
```

Comment:
Why would someone want to do this? Can you add a brief note describing the purpose of this step, and indicating whether it's optional?